invenio_app_ils/ui/backoffice/src/pages/ItemsSearch/ItemsSearch.scss

```diff
@@ -3,8 +3,13 @@
 }
 
 .items-search-container {
-  .results-per-page {
-    padding: 0 0.5em;
+  .n-per-page-sorting {
```

Comment:
Omit n? I would wait to be a function that you pass the n value and return a dynamic class...

Reply:
Done!